packages/edit-site/src/hooks/use-theme-style-variations/use-theme-style-variations-by-property.js

```diff
@@ -47,17 +47,31 @@ export function removePropertyFromObject( object, property ) {
 	return object;
 }
 
+/**
+ * Checks whether a style variation is empty.
+ *
+ * @param {Object} variation          A style variation object.
+ * @param {string} variation.title    The title of the variation.
+ * @param {Object} variation.settings The settings of the variation.
+ * @param {Object} variation.styles   The styles of the variation.
+ * @return {boolean} Whether the variation is empty.
+ */
+function isEmptyStyleVariation( { title, settings, styles } ) {
+	return (
+		title === __( 'Default' ) || // Always preseve the default variation.
```

Comment:
```suggestion
		title === __( 'Default' ) || // Always preserve the default variation.
```

Reply:
Why is default always an empty style variation?

Reply:
Default removes all changes to the user global styles so that it reverts back to whatever theme.json is setting.